native/src/main/java/io/ballerina/stdlib/kafka/utils/KafkaUtils.java

```diff
@@ -484,13 +484,14 @@ private static void addStringOrStringArrayParamIfPresent(String paramName,
                                                      Properties configParams,
                                                      BString key) {
         if (configs.containsKey(key)) {
-            List<String> values;
+            ArrayList<String> values;
```

Comment:
You don't have to change this, for the changes you have done in lines 493 & 4.

Reply:
These are reverted and added a new fix